### ISSUE TITLE
Forum Permission Download Thread

### ISSUE THREAD

**Issue by dragonexpert** [dragonexpert/downloadthread#12]
I think it might be good to have downloading thread be able to be controlled on a per forum basis besides just the general group permission.  This way a VIP group might be able to download a thread from one forum, but a regular user could not.

**Comment by lairdshaw** [comment]
So, are you saying that you want to change the logic of these lines?:

https://github.com/dragonexpert/downloadthread/blob/1364bc604d937bb1a5fde78c75637442c42904d8/inc/plugins/downloadthread/hooks.php#L27-L34

**Comment by dragonexpert** [comment]
Yes.  This would be more in line with how permissions are handled in core.  The usergroup has a default.  Forum permissions can overide the default.
